samples/cpp/rag_sample/include/db_pgvector.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef _DBPGVECTOR
+#define _DBPGVECTOR
+
+#include "state.hpp"
+
+// pgvector-cpp
+#include <iostream>
+#include <pqxx/pqxx>
+#include "pqxx.hpp"
+#include <cassert>
+#include <optional>
+
+class DBPgvector{
+    public:
+
+        pqxx::connection pgconn{"user=postgres host=localhost password=openvino port=5432 dbname=postgres"};
```

Comment:
We can move this init DB to de_setup() with host, passwd, port, dbname as input.